care/emr/resources/patient/spec.py

```
@@ -62,6 +89,20 @@ def validate_geo_organization(cls, geo_organization):
 
+    @model_validator(mode="after")
+    def validate_identifiers(self):
+        instance_identifier_configs = PatientIdentifierConfigCache.get_instance_config()
+        configs = {str(x.config): x for x in self.identifiers}
+        for identifier_config in instance_identifier_configs:
+            if identifier_config["id"] in configs:
+                value = configs[identifier_config["id"]].value
+                validate_identifier_config(identifier_config, value)
+            elif identifier_config["config"]["required"]:
+                raise ValueError(
+                    f"Identifier config {identifier_config['config']['system']} is required"
```

Ruff (0.11.9)

Exception must not use an f-string literal, assign to variable first

Assign to variable; remove f-string literal

(EM102)